buildlib/pr/main.yml

```diff
@@ -173,3 +176,17 @@ stages:
         parameters:
             name: gpu
             demands: ucx_gpu -equals yes
+
+  - stage: python
+    dependsOn: [Static_check]
+    jobs:
+      - template: python/python-test.yml
+        parameters:
+          name: new
+          demands: ucx_new -equals yes
```

Comment:
make sure that docker installed for `new` host

Reply:
I also don't know who should handle this.